### ISSUE TITLE
how to improve the Accuracy by fine-tuning the parameter of gcForest？

### ISSUE THREAD

**Issue by Irving-ren** [kingfengji/gcForest#50]
Hi, I am sorry about the previous question. the gcForest was working well under my environment. However,  I am confused about the result after using gcForest to handle the multi-classification problem. Here is the related code of my issue. 

     def parse_args():
        parser = argparse.ArgumentParser()
        parser.add_argument("--model", dest="model", type=str, default=None, help="gcfoest Net Model File")
        args = parser.parse_args()
        return args

    def get_toy_config():
    config = {}
    ca_config = {}
    ca_config["random_state"] = 0
    ca_config["max_layers"] = 100
    ca_config["early_stopping_rounds"] = 3
    ca_config["n_classes"] = 3
    ca_config["estimators"] = []
    ca_config["estimators"].append(
            {"n_folds": 5, "type": "XGBClassifier", "n_estimators": 10, "max_depth": 5,
             "objective": "multi:softprob", "silent": True, "nthread": -1, "learning_rate": 0.1} )
    ca_config["estimators"].append({"n_folds": 5, "type": "RandomForestClassifier", "n_estimators": 10, "max_depth": None, "n_jobs": -1})
    ca_config["estimators"].append({"n_folds": 5, "type": "ExtraTreesClassifier", "n_estimators": 10, "max_depth": None, "n_jobs": -1})
    ca_config["estimators"].append({"n_folds": 5, "type": "LogisticRegression"})
    config["cascade"] = ca_config
    return config

In my case, I just change the number of classes to three. BTW, the input vector was  the matrix（1280*320)， and the labeled  data was the matrix（1280,) . It turns out the accuracy of leave-one-group-out was just like this.
![screenshot from 2018-10-17 10-14-37](https://user-images.githubusercontent.com/13758812/47056049-80a09380-d1f5-11e8-9f33-79f7f5eae65e.png)
And, I used the MLP for my data also. the result is much better than gcForest. Do you have any clue for this problem？maybe the hyper-parameter of gcForest？ Thanks for your patience.
Best regards
Irving
